### PR TITLE
Hotfix: Proeminent disclosure update

### PR DESCRIPTION
**Pull request by Te-Z**
Prominent disclosure updated to be closer to Google requirements. 

More informations: https://support.google.com/googleplay/android-developer/answer/10144311?visit_id=638573089252378170-2363916442&rd=1#

TLDR: 

> To meet policy requirements, it’s recommended that you reference the following example format for Prominent Disclosure when it’s required:
> 
> “[This app] collects/transmits/syncs/stores [type of data] to enable ["feature"], [in what scenario]."
> Example: “Fitness Funds collects location data to enable fitness tracking even when the app is closed or not in use and is also used to support advertising.”
> Example: “Call buddy collects read and write call log data to enable contact organization even when the app is not in use.”

https://github.com/user-attachments/assets/0c1c4353-510d-445e-ac17-e1ecf817e924

https://github.com/user-attachments/assets/ec999248-2e99-48b3-86e6-168972991ddb

### CODE REVIEW

File: lib/presentation/mixins/media_picker_mixin.dart [hunk 1]
```diff
@@ -288,6 +293,11 @@ mixin MediaPickerMixin on CommonMediaPickerMixin {
           ),
         ),
       ),
+      onGoToSettings: (ctx) async {
+        Navigator.pop(ctx);
```
Comment (nqhhdev) [inline lib/presentation/mixins/media_picker_mixin.dart:297]:
```suggestion
      onGoToSettings: (context) async {
        Navigator.pop(context);
```

**Comment by nqhhdev** [comment]
Version app: 2.6.0
Device: iOS - version 17.0

I found issue related to permission. When I allow native permission → UI can't rebuild.

Pls fix it on your PR


https://github.com/user-attachments/assets/05bdbcff-da0e-4fb5-8e9b-4ff2639bde71

File: lib/presentation/mixins/media_picker_mixin.dart [hunk 2]
```diff
@@ -44,21 +48,22 @@ mixin MediaPickerMixin on CommonMediaPickerMixin {
...
-        onSendTap: onSendTap,
-        room: room,
-        onPickerTypeTap: onPickerTypeTap,
-        onCameraPicked: onCameraPicked,
-        focusSuggestionController: focusSuggestionController,
-        captionController: captionController,
-        typeAheadKey: typeAheadKey,
-      );
-    }
+    await getCurrentMediaPermission(context)?.then((currentPermissionPhotos) {
```
Comment (sherlockvn) [inline lib/presentation/mixins/media_picker_mixin.dart:51]: add the `catchError` also, just in case we have error

**Comment by Te-Z** [comment]
> Version app: 2.6.0 Device: iOS - version 17.0
> 
> I found issue related to permission. When I allow native permission → UI can't rebuild.
> 
> Pls fix it on your PR
> 
>  Simulator.Screen.Recording.-.iPhone.15.-.2024-07-23.at.23.07.47.mp4

Hi @nqhhdev I won't have enough time this morning to fix this PR, could you help me please ?

**Comment by nqhhdev** [comment]
> > Version app: 2.6.0 Device: iOS - version 17.0
> > I found issue related to permission. When I allow native permission → UI can't rebuild.
> > Pls fix it on your PR
> > Simulator.Screen.Recording.-.iPhone.15.-.2024-07-23.at.23.07.47.mp4
> 
> Hi @nqhhdev I won't have enough time this morning to fix this PR, could you help me please ?

I fixed 


https://github.com/user-attachments/assets/830da695-79a6-4060-b552-7eadb6d5f49d


https://github.com/user-attachments/assets/1fc1c244-8d2a-4792-9617-b1e4684ccce2

**Comment by Te-Z** [comment]
tested on android LGTM

**Review by sherlockvn — APPROVED** [review]

**Pull request merged**